proposal.md

```diff
@@ -111,10 +111,18 @@ SEE ALSO (map to relevant subsections):
 #### Help pages
 
 Source files for help pages use the "R documentation" (Rd) format. 
-`utils::prompt` and `utils::promptData` may be used to create an Rd template for a function or data set, respectively. `tools::Rdcheck` may be used to validate Rd files, e.g., detecting syntax errors.
+`utils::prompt()` and `utils::promptData()` may be used to create an Rd template for a function or data set, respectively. `tools::checkRd()` may be used to validate Rd files, e.g., detecting syntax errors.
 
 WRE reference: [Writing R documentation files](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Writing-R-documentation-files)
 
+- `r pkg("roxygen2", priority = "core")` provides the `roxygenise()` function to generate Rd files from roxygen blocks: comments with specific markup in R source files. When used with Markdown, roxygen can greatly reduce the markup required in the documentation source. Several [development workflow](#development-workflow) packages support creating documentation with roxygen. `r pkg("Rdroxygen")` provides functions to convert Rd files to roxygen blocks; `r pkg("roxygen2md")` replaces Rd syntax in roxygen comments with the Markdown equivalent.
+- `r pkg("sinew")` generates roxygen skeletons and updates the NAMESPACE and DESCRIPTION file as required; can be used to update as well as create royxgen documentation.  
+- `r pkg("roclang")` facilitates extracting components of documentation from an Rd file (in the same or another package) for reuse, by insertion in a roxygen comment.
+- `r pkg("pasteAsComment")` provides an RStudio addin to paste clipboard content as a roxygen block - useful for inserting examples; `r github("csgillespie/roxygen2Comment")` provides an RStudio addin to convert regular code to roxygen commented code and vice versa.
+- `r pkg("roxyglobals")` adds additional roxygen tags to generate R code defining global variables with `utils::globalVariables()`. This can be used to avoid false positives in the package check when functions in the package call functions that use non-standard evaluation.
+- `r pkg("Rdpack")` provides functions and Rd macros for developing documentation, e.g. adding template documentation for new arguments; importing references from BibTeX files, and evaluating R code then inserting the resulting output or graphic.
+- `r github("Genentech/rd2markdown")` converts Rd files to plain Markdown files; `r github("coolbutuseless/rd2list")` converts Rd files to an R list.
+
```

Comment:
Some other packages, many from @maelle [blog post](https://masalmon.eu/2024/09/03/roxygen2-custom-tag/).
Perhaps [devtag](https://github.com/moodymudskipper/devtag) is a package also worth mentioning. It helps with help pages only intended for developers.
There is also [roxytest](https://cran.r-project.org/package=roxytest) that documents and creates test for developers (I mention it on the testing section but might be worth mentioning it here too)

Reply:
I added both devtag and srr. I think we can keep roxytest in the test section.